src/test/components/Day/Day.test.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { List, ListItem, ListSubHeader} from 'react-toolbox/lib/list'
 import { shallow } from 'enzyme';
+import renderer from 'react-test-renderer';
 
 import Day from '../../../components/Day/Day'
```

Comment:
Why not an absolute import? Our Webpack config should treat the `src/` as a root dir.